packages/wasm-utils/src/note/mod.rs

```diff
@@ -1,20 +1,22 @@
-use core::fmt;
-use std::str::FromStr;
-
 use arkworks_setups::common::Leaf;
+use core::fmt;
 use js_sys::{JsString, Uint8Array};
 use rand::rngs::OsRng;
+use std::str::FromStr;
```

Comment:
We should avoid std lib right?

Reply:
We had it from the beginning, if we should use a non-std option what do you recommend?

Reply:
`core::str::FromStr`?